doc/tutorials/run_kata_containers.rst

```diff
@@ -0,0 +1,329 @@
+   $ cd $(go env GOPATH)/src/github.com/kata-containers/shim && make && sudo make install
+   $ cd $(go env GOPATH)/src/github.com/kata-containers/agent && make && sudo make install
+
+Configure Docker
+================
+
+The block-based driver like device-mapper is required for running Kata-containers on ACRN.
+
+Configure a Docker proxy
+------------------------
+
+If you use a proxy server, run the following commands as a shell script to configure Docker,
+refer to this `tutorial <https://docs.01.org/clearlinux/latest/tutorials/kata.html#run-kata-containers>`_
+for detailed information:
+
+.. code-block:: bash
+
+   export http_proxy=<your http proxy>:<port>
+   export https_proxy=<your https proxy>:<port>
+   sudo mkdir -p /etc/systemd/system/docker.service.d/
+   cat <<EOF | sudo tee /etc/systemd/system/docker.service.d/50-runtime.conf
+   [Service]
+   Environment="DOCKER_DEFAULT_RUNTIME=--default-runtime kata-runtime"
+   EOF
+   docker_service_dir="/etc/systemd/system/docker.service.d/"
+   sudo mkdir -p "$docker_service_dir"
+   cat <<EOF | sudo tee "$docker_service_dir/proxy.conf"
+   [Service]
+   Environment="HTTP_PROXY=$http_proxy"
+   Environment="HTTPS_PROXY=$https_proxy"
+   EOF
+   echo "Reloading unit files and starting docker service"
+   sudo systemctl daemon-reload
+   sudo systemctl restart docker
+   sudo docker info
+
+Enable the "devicemapper" storage driver
+----------------------------------------
+
+#. Stop Docker daemon if it is already running::
+
+   $ sudo systemctl stop docker
+
+#. Specify the storage driver to "devicemapper":
+
+   .. code-block:: bash
+
+      cat <<EOF | sudo tee /etc/docker/daemon.json
+      {
+      "storage-driver": "devicemapper"
+      }
+      EOF
+
+#. Reload daemon and the restart the docker to enable the new storage driver:
+
+   .. code-block:: bash
+
+      $ sudo systemctl daemon-reload
+      $ sudo systemctl restart docker
+      $ sudo docker info
+
+#. You should see "Storage Driver: devicemapper" from the ``docker info``.
+
+Add kata-runtime to the Docker server
+-------------------------------------
+
+Add ``kata-runtime`` to docker:
+
+.. code-block:: none
+
+   $ sudo systemctl stop docker
+   $ dockerUnit=$(systemctl show -p FragmentPath docker.service | cut -d "=" -f 2)
+   $ unitFile=${dockerUnit:-/etc/systemd/system/docker.service.d/kata-containers.conf}
+   $ test -e "$unitFile" || { sudo mkdir -p "$(dirname $unitFile)"; echo -e "[Service]\nType=simple\nExecStart=\nExecStart=/usr/bin/dockerd -D --default-runtime runc" | sudo tee "$unitFile"; }
+   $ grep -q "kata-runtime=" $unitFile || sudo sed -i 's!^\(ExecStart=[^$].*$\)!\1 --add-runtime kata-runtime=/usr/local/bin/kata-runtime!g' "$unitFile"
+   $ sudo systemctl daemon-reload
+   $ sudo systemctl restart docker
+
+Use ``sudo docker info`` to check the ``kata-runtime`` is added correctly.
+
+.. code-block:: console
+   :emphasize-lines: 1
+
+   Runtimes: kata-runtime runc
+   Default Runtime: runc
```

Comment:
This is not correct as you instruct users to make `kata-runtime` the default above.

Reply:
Remove the default runtime setup paragraph above and only focus on `Runtimes: kata-runtime` is set up here.